### PR TITLE
quick fix kill command under windows

### PR DESCRIPTION
**Pull request by J-shang**

### CODE REVIEW

File: ts/nni_manager/main.ts [hunk 1]
```diff
@@ -204,4 +191,10 @@ process.on(getStopSignal(), async () => {
         await log.close();
         process.exit(hasError ? 1 : 0);
     }
-});
+}
+
+process.on('SIGTERM', cleanUp);
+process.on('SIGBREAK', cleanUp);
+process.on('SIGINT', cleanUp);
+//not sure if 'SIGKILL' signal can be listened
```
Comment (liuzhe-lz) [inline ts/nni_manager/main.ts:199]: It can't.

Comment (J-shang) [reply ts/nni_manager/main.ts]: fix it

**Comment by SparkSnail** [comment]
@liuzhe-lz So change back to `CTRL_BREAK_EVENT` is ok?

File: ts/nni_manager/main.ts [hunk 2]
```diff
@@ -201,7 +188,11 @@ process.on(getStopSignal(), async () => {
         hasError = true;
         log.error(`${err.stack}`);
     } finally {
-        await log.close();
+        log.close();
```
Comment (SparkSnail) [inline ts/nni_manager/main.ts:191]: why remove await?

Comment (J-shang) [reply ts/nni_manager/main.ts:191]: `Logger.close()` is a sync function

**Review by SparkSnail — APPROVED** [review]

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**